File: eval/fixtures/cross-repo/acme-core/src/task.ts
export interface Task {
  id: string;
  title: string;
  dueAt: number;
  done: boolean;
}

export function createTask(id: string, title: string, dueAt: number): Task {
  if (title.trim() === '') throw new Error('title must not be empty');
  return { id, title, dueAt, done: false };
}

/** A task is overdue when it is still open and its due date has passed. */
export function overdue(task: Task, now: number): boolean {
  return !task.done && task.dueAt < now;
}

File: eval/fixtures/cross-repo-py/py-core/taskcore/task.py
"""Task domain type and the pure helpers over it."""
from dataclasses import dataclass


@dataclass
class Task:
    id: str
    title: str
    due_at: int
    done: bool = False


def create_task(id: str, title: str, due_at: int) -> Task:
    if title.strip() == "":
        raise ValueError("title must not be empty")
    return Task(id=id, title=title, due_at=due_at)


def overdue(task: Task, now: int) -> bool:
    """A task is overdue when it is still open and its due date has passed."""
    return not task.done and task.due_at < now
